arm9/source/game/nds.c

```diff
@@ -133,7 +133,9 @@ u32 GetTwlTitle(char* desc, const TwlIconData* twl_icon) {
 u32 GetTwlIcon(u16* icon, const TwlIconData* twl_icon) {
     const u32 h = TWLICON_DIM_ICON; // fixed size
     const u32 w = TWLICON_DIM_ICON; // fixed size
-    const u16* palette = twl_icon->palette;
+    u16* palette = twl_icon->palette;
+    u16 palette_first = palette[0];
+    palette[0] = 0x7FFF;
```

Comment:
You shouldn't write here, even if you set the value back later on, there's a better solution....

Reply:
I did that before, but @Wolfvak recommended to do this instead:
```
<Wolfvak> you can restore the original value after you're done?
<Wolfvak> it's much faster than branching
```

Reply:
Yeah, I messed up and didn't notice it was `const`. If you wanna keep this approach you'd have to copy the palette to a local buffer.
I'm not sure if it's worth it, when comparing the setup time of this temp buffer to the time needed by the conversion time itself.

Reply:
@d0k3 I reverted that change and went back to my original method, which is the way you suggested.